Source: Timo Gurr <tgurr@exherbo.org>
Upstream: No
Reason: Allow two additional tests to (x-)fail

diff -Naur glibc-2.21.orig/conform/Makefile glibc-2.21/conform/Makefile
--- glibc-2.21.orig/conform/Makefile	2015-02-06 07:40:18.000000000 +0100
+++ glibc-2.21/conform/Makefile	2015-02-12 15:33:41.298755929 +0100
@@ -350,6 +350,7 @@
 test-xfail-XPG3/regex.h/linknamespace = yes
 test-xfail-XPG3/search.h/linknamespace = yes
 test-xfail-XPG3/stdio.h/linknamespace = yes
+test-xfail-XPG3/stdlib.h/linknamespace = yes
 test-xfail-XPG3/unistd.h/linknamespace = yes
 test-xfail-XPG3/wordexp.h/linknamespace = yes
 test-xfail-XPG4/fmtmsg.h/linknamespace = yes
@@ -413,5 +414,6 @@
 test-xfail-XOPEN2K8/regex.h/linknamespace = yes
 test-xfail-XOPEN2K8/search.h/linknamespace = yes
 test-xfail-XOPEN2K8/spawn.h/linknamespace = yes
+test-xfail-XOPEN2K8/stdlib.h/linknamespace = yes
 test-xfail-XOPEN2K8/syslog.h/linknamespace = yes
 test-xfail-XOPEN2K8/unistd.h/linknamespace = yes
